grid2op/Opponent/OpponentSpace.py

```diff
@@ -31,7 +31,8 @@ class OpponentSpace(object):
     budget_per_timestep: ``float``
         The increase of the opponent budget per time step (if any)
     """
-    def __init__(self, compute_budget, init_budget, opponent, budget_per_timestep=0., action_space=None):
+    def __init__(self, compute_budget, init_budget, opponent, attack_duration, attack_cooldown,
```

Comment:
is this properly updated when it's created in the environment ?

Reply:
Yes, I think so :
```
self.oppSpace = OpponentSpace(compute_budget=self.compute_opp_budget,
                                init_budget=self.opponent_init_budget,
                                attack_duration=self.opponent_attack_duration,
                                attack_cooldown=self.opponent_attack_cooldown,
                                budget_per_timestep=self.opponent_budget_per_ts,
                                opponent=self.opponent
                                )
```